server/src/main/java/org/opensearch/node/Node.java

```diff
@@ -424,6 +426,13 @@ protected Node(
                 initialEnvironment.pluginsDir(),
                 classpathPlugins
             );
+
+            if (FeatureFlags.isEnabled(FeatureFlags.EXTENSIONS)) {
+                this.extensionsOrchestrator = new ExtensionsOrchestrator(tmpSettings, initialEnvironment.extensionDir());
+            } else {
+                this.extensionsOrchestrator = null;
```

Comment:
`Noop` would eliminate dealing with `null` everywhere

Reply:
What would the replacement line be instead of null?

Reply:
Something like `extensionsOrchestrator = new NoopOrchestrator()`